phpunit.xml.dist

```diff
@@ -0,0 +1,16 @@
+<phpunit bootstrap="vendor/silverstripe/cms/tests/bootstrap.php" colors="true">
```

Comment:
I see you've got the CMS recipe in the travis config, is it a requirement for testing? Or could this reference `framework` instead? No need for CMS jumped out at me while reading through.

Reply:
I put it there because there’s a Page extension in this class, which isn’t tested yet, but I had some thoughts about adding tests for it